py3status/modules/pomodoro.py

```diff
@@ -271,27 +284,33 @@ def pomodoro(self):
                 vals['mmss'] = u'%d%s%02d' % (
                     minutes, self.timer_separator, seconds)
 
-        if self.py3.format_contains(self.format, 'bar'):
+        # turn on bar?
+        if self.py3.format_contains(self.format_timer, 'bar'):
             vals['bar'] = self._setup_bar()
 
+        # response
+        full_text = self.py3.safe_format(self.format_timer, vals)
+        cached_until = 0
         if self._running:
-            format = u'{{prefix}} [{}]'.format(self.format)
+            full_text = self.py3.safe_format(self.format_start, {'timer': full_text})
         else:
-            format = u'{{prefix}} ({})'.format(self.format)
+            full_text = self.py3.safe_format(self.format_stop, {'timer': full_text})
             cached_until = self.py3.CACHE_FOREVER
 
         response = {
-            'full_text': format.format(prefix=self._prefix, **vals),
+            'full_text': self.py3.safe_format(self.format, {'timer': full_text}),
             'cached_until': cached_until,
```

Comment:
`full_test` needs to be a composite.  The easy way is to use `py3.composite_create(full_text)`.

That way the formatter knows that this is a composite not just a list or dict